### DISCUSSION TITLE
Re-labling of raised issue - #407

### DISCUSSION THREAD

**Discussion by Nishant-k-sagar** [MAVRICK-1/e-commerce_website#408] [Q&A]
in recent, i have raised an issue regarding UI improvement of Sign-in Page, but somehow i missed to add lables, how can i re-lable that issue?or i have to raise a new issue regarding same? -#407

**Comment by MAVRICK-1** [comment]
@Nishant-k-sagar contributors can't re-assign the labels, you can only assign it once you open an issue
